packages/thirdweb/src/utils/uuid.ts

```diff
@@ -1,18 +1,8 @@
-import { universalCrypto } from "../crypto/aes/lib/universal-crypto.js";
 import { uint8ArrayToHex } from "./encoding/hex.js";
 
-/**
- * @internal
- */
-export async function uuid() {
-  return (await universalCrypto()).randomUUID();
-}
-
 /**
  * @internal
  */
 export async function randomBytes32() {
-  return uint8ArrayToHex(
-    (await universalCrypto()).getRandomValues(new Uint8Array(32)),
-  );
+  return uint8ArrayToHex(globalThis.crypto.getRandomValues(new Uint8Array(32)));
```

Comment:
this *should* be fine but if this PR breaks anything this will be the culprit

Reply:
i'll run the auth template with a dev release to check